client/components/chart/index.jsx

```diff
@@ -5,15 +5,18 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
-import { noop, throttle } from 'lodash';
+import { noop } from 'lodash';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import BarContainer from './bar-container';
+import afterLayoutFlush from 'lib/after-layout-flush';
```

Comment:
TIL! And really enjoyed https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Performance_best_practices_for_Firefox_fe_engineers